app/views/spree/admin/payments/show.html.haml

```diff
@@ -1,4 +1,5 @@
-= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_page_title', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
 
 - content_for :page_title do
```

Comment:
doesn't this conflict with the order_page_title partial?

Reply:
This PR did get a bit out of order with commits, so I've made the necessary changes and will push now. I believe this code is okay as it was not changed by me previously.